deribit-api/src/main/java/io/contek/invoker/deribit/api/websocket/WebSocketApi.java

```diff
@@ -0,0 +1,41 @@
+package io.contek.invoker.deribit.api.websocket;
+
+import com.google.common.collect.ImmutableList;
+import io.contek.invoker.commons.actor.IActor;
+import io.contek.invoker.commons.actor.ratelimit.RateLimitQuota;
+import io.contek.invoker.commons.websocket.*;
+import io.contek.invoker.deribit.api.websocket.common.WebSocketInfoMessage;
+import io.contek.invoker.security.ICredential;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+@ThreadSafe
+public abstract class WebSocketApi extends BaseWebSocketApi {
+
+  private final WebSocketContext context;
+
+  protected WebSocketApi(IActor actor, WebSocketContext context) {
+    super(actor, WebSocketMessageParser.getInstance(), IWebSocketAuthenticator.noOp());
+    this.context = context;
+  }
+
+  @Override
+  protected ImmutableList<RateLimitQuota> getRequiredQuotas() {
+    return ImmutableList.of();
+  }
+
+  @Override
+  protected WebSocketCall createCall(ICredential credential) {
+    return WebSocketCall.fromUrl(context.getBaseUrl() + "/ws/api/v2");
+  }
+
+  @Override
+  protected final void checkErrorMessage(AnyWebSocketMessage message) {
+    if (message instanceof WebSocketInfoMessage) {
```

Comment:
I believe deribit does not have such message type. Delete this class and remove any logic corresponds to this type.